backend/inspirehep/oai/serializers.py

```diff
@@ -0,0 +1,14 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2020 CERN.
+#
+# inspirehep is free software; you can redistribute it and/or modify it under
+# the terms of the MIT License; see LICENSE file for more details.
+
+from inspire_dojson.api import record2marcxml
+from lxml import etree as ET
+
+
+def record_json_to_marcxml(pid, record, **kwargs):
+    """Converts record to marcxml for OAI."""
+    return ET.fromstring(record2marcxml(record["_source"]))
```

Comment:
you're supposed to return a tree here and not a string? maybe we should add a function for that in the inspire-dojson API, as now it's a bit silly, we serialize in https://github.com/inspirehep/inspire-dojson/blob/master/inspire_dojson/api.py#L142 then immediately deserialize here.

Reply:
yes, it needs a tree. Let's for know merge it and if everything is ok I'll do the dance with dojson etc